include/nuttx/sensors/sensor.h

```diff
@@ -495,6 +495,58 @@ struct sensor_ops_s
 
   CODE int (*batch)(FAR struct sensor_lowerhalf_s *lower,
                     FAR unsigned int *latency_us);
+
+  /**************************************************************************
+   * Name: fetch
+   *
+   * We can fetch sensor register data by this function. It will use buffer
+   * of sensor_read provided and disables intermediate buffer of upper half.
+   * We recommend lowerhalf driver writer to use this function for slower
+   * ODR(output data rate) of sensor because this way saves space and simple.
```

Comment:
```suggestion
   * It is recommended that the lowerhalf driver writer use this function for slower
   * sensor ODR (output data rate) because this way saves space and it's simpler.
```

Reply:
done